build/target/product/aosp_x86_64.mk

```diff
@@ -28,34 +28,43 @@ PRODUCT_USE_DYNAMIC_PARTITIONS := true
 # build quite specifically for the emulator, and might not be
 # entirely appropriate to inherit from for on-device configurations.
 
-# GSI for system/product
+#
+# All components inherited here go to system image
+#
 $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit.mk)
-$(call inherit-product, $(SRC_TARGET_DIR)/product/gsi_common.mk)
+$(call inherit-product, $(SRC_TARGET_DIR)/product/mainline_system.mk)
```

Comment:
I don't understand, if `mainline_system.mk` was deleted, then why is it referenced here?

Reply:
The name `media_system.mk` as mistakenly exchanged with `mainline_system.mk`, will be fixed.